### ISSUE TITLE
Add support for .pssproj

### ISSUE THREAD

**Issue by isfr** [fernandoescolar/vscode-solution-explorer#64]
I have some PowerShell projects created from Visual Studio, and it would be awesome if you could add support for them in your extension.

**Comment by Pwd9000-ML** [comment]
Yes please, I am also having issues with this one as I am a PowerShell module developer and the extension is pretty cool but lacks support for pssproj. Thumbs up here for the enhancement!
